homeassistant/components/openuv/coordinator.py

```diff
@@ -116,6 +116,7 @@ async def _async_update_data(self) -> dict[str, Any]:
             data = await self.update_method()
         except InvalidApiKeyError:
             await self._invalid_api_key_monitor.async_increment()
+            self.last_update_success = False
```

Comment:
Shouldn't it raise instead?

Reply:
Yes—not sure what I was thinking there. 😂